### ISSUE TITLE
Creating database with Docker via Powershell ISE

### ISSUE THREAD

**Issue by armela1981** [Microsoft/nav-docker#201]
Hello there.
I am able to use the Powershell ISE and run the command to download all the files.
Seems like everything works well, however I can not open the Cside, RTC or web client (shortcuts created in  my desktop).
Here is what  my screenshot looks like , however the server name MyBizCentralApp\SQLEXPRESS does no not exist. Do I need to do anything with setting up a server and service tier ? I thought the docker image did all of that when it created the shortcuts.  Thanks

![image](https://user-images.githubusercontent.com/39662533/40677403-2bab1c36-6343-11e8-8454-983c032d75c9.png)

**Comment by freddydk** [comment]
new-navcontainer creates the container, the shortcuts and all
remove-navcontainer removes everything
if you have shortcuts on the desktop not pointing to anything it is likely because you have removed the container using other commands - then remove them manually.
If you are creating the container using other commands you get no new shortcuts

**Comment by armela1981** [comment]
See screenshot below. I can run this many times and it removes and re-creates the desktop shortcuts, but I can't still get to the Cside or RTC database.
Any ideas ??

![image](https://user-images.githubusercontent.com/39662533/40678201-7758f2be-6345-11e8-8f90-6f2c64979859.png)

**Comment by freddydk** [comment]
Are you entering the windows credentials on the machine on which you are running?
You can also change to Username/Password auth by changing -auth Windows to -auth NavUserPassword

**Comment by freddydk** [comment]
Did you ever manage to make this work?

**Comment by armela1981** [comment]
Yes I did. I ended up changing to user name/password instead of windows auto. Thank u.

Best Regards

On Jun 4, 2018, at 5:52 PM, Freddy Kristiansen <notifications@github.com<mailto:notifications@github.com>> wrote:


Did you ever manage to make this work?

-
